### PR TITLE
adds sex and age data for united healthcare brfss data

### PR DESCRIPTION
**Pull request by joshzarrabi**
- calculates the correct age buckets for acs in the backend

### CODE REVIEW

**Review by vanshkumar — COMMENTED** [review]
Do you know why age is not working?

image
image

**Comment by joshzarrabi** [comment]
> Do you know why age is not working?
> 
> image
> image

I need to merge the pr and run the airflow for the data to be in the bucket

File: frontend/src/data/react/WithLoadingOrErrorUI.test.tsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ describe("WithLoadingOrErrorUI", () => {
     act(() => {
       dataFetcher.setFakeMetadataLoaded(fakeMetadata);
       dataFetcher.setFakeDatasetLoaded("brfss", []);
-      dataFetcher.setFakeDatasetLoaded("acs_population-by_age_county", []);
+      dataFetcher.setFakeDatasetLoaded("uhc_race_and_ethnicity", []);
```
[the remark this answers is not in the document]
Comment (joshzarrabi) [reply frontend/src/data/react/WithLoadingOrErrorUI.test.tsx:129]: omg yeah, weird the test is passing .....

File: config/data_sources/AGG_acs_population.sql [hunk 2]
```diff
@@ -63,6 +88,23 @@ SELECT * EXCEPT(sex)
...
+SELECT * EXCEPT(sex)
+FROM `acs_population.by_sex_age_state_big`
+WHERE sex = "Total";
+
+CREATE OR REPLACE TABLE acs_population.by_age_county_big AS
+SELECT * EXCEPT(sex)
+FROM `acs_population.by_sex_age_county_big`
+WHERE sex = "Total";
+
+CREATE OR REPLACE TABLE acs_population.by_age_state AS
```
[the remark this answers is not in the document]
Comment (joshzarrabi) [reply config/data_sources/AGG_acs_population.sql:101]: I didn't know of another way to merge the tables

[the remark this answers is not in the document]
Comment (vanshkumar) [reply config/data_sources/AGG_acs_population.sql:101]: Sorry yeah that's my bad, I misread this line

**Comment by vanshkumar** [comment]
> > Do you know why age is not working?
> > image
> > image
> 
> I need to merge the pr and run the airflow for the data to be in the bucket @vanshkumar

I see, SG

**Review by vanshkumar — APPROVED** [review]

**Pull request merged**